app/src/main/res/layout/layout_ai_engine_choice.xml

```diff
@@ -46,5 +46,5 @@
         android:layout_height="wrap_content"
         android:layout_margin="@dimen/marginStandardMedium"
         app:icon="@drawable/ic_external_information"
-        app:text="@string/aiEngineWarning" />
+        app:title="@string/aiEngineWarning" />
```

Comment:
The already existing InformationBlockView should use the description to stay coherent with the previous style